packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -127,9 +127,21 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
     if (!dom) {
       return;
     }
-
+    let isTestEnv = process.env.NODE_ENV === 'test';
     let w = dom.clientWidth;
     let h = dom.clientHeight;
+    // console.log('in scrollview', Object.getOwnPropertyNames(window.HTMLElement.prototype), Object.getOwnPropertyNames(window.HTMLElement.prototype).includes('clientHeight'), Object.getOwnPropertyDescriptor(window.HTMLElement.prototype, 'clientHeight'))
+    // TODO: set w and h to Infinity so the ScrollView can render any amount of content in a non-mocked test env ScrollView
+    // TODO: make condition here the following: is Test env (process.env) and clientWidth/Height aren't defined? That way
+    // someone can still mock it
+    // TODO: the below isn't technically needed if we just mock getVisibleLayoutInfos in Virtualizer but then we won't be able to
+    // check if the clientWidth/Height is being mocked?
+    if (isTestEnv) {
+      w = Infinity;
+      h = Infinity;
+      // TODO: maybe make w/h equal to the content width and height aka always fit?
+    }
```

Comment:
ideally we'd still test the virtualized behavior in RSP tests, still trying to figure out a good way to determine if clientHeight/Width has been mocked or not without the tests blowing up